src/utils/Validations.ts

```diff
@@ -68,3 +69,7 @@ export const isPersonNameValid = (personName: PersonIdentityName[]) : boolean =>
 	}
 	return isValid;
 };
+
+export const isValidUUID = (code: string): boolean => {
```

Comment:
Would you mind adding a couple of unit tests for this please?

Reply:
Added.